### ISSUE TITLE
RSSI Low event stays on only once and for short time only

### ISSUE THREAD

**Issue by Alex00101** [FrSkyRC/ETHOS-Feedback-Community#3902]
I think there was another issue (the warning payed once - #3882) so this may be a duplicate or rather a bit of a deep dive. 
Effectively once rssi level low is reached, system event SYSTEM_EVENT_RSSI_LOW stays on only for the short amount of time while the rssi low warning is played, after that it auto resets to off.

To reproduce:
- a widget that constantly prints a rawValue of the source  system.getSource({category=CATEGORY_SYSTEM_EVENT, member=SYSTEM_EVENT_RSSI_LOW}) from wakeup method
- In a simulator, set RSSI low warning to say 67db (as simulator outputs a constant value of 65db)
- Observe an output, the value returns 100 (On) only while the warning is played then it gets back to -100

--------------------------------        -100
--------------------------------        -100
--------------------------------        -100
--------------------------------        -100
002338 playFile('/audio/en/ethos/system/rssi-int-low-warning.wav', flags=0, id=20)
002338 playHaptic(duration=10)
**--------------------------------        100
--------------------------------        100
--------------------------------        100
--------------------------------        100**
--------------------------------        -100
--------------------------------        -100
--------------------------------        -100
--------------------------------        -100

I'd imagine that the event should stay on for as long as the RSSI is below the low warning threshold?

Thanks

**Comment by bsongis-frsky** [comment]
True. I have to review all those System events. Today they are reset after 100ms, which could be wrong.

**Comment by Alex00101** [comment]
Thank you, much appreciated. These events are (at least for me as a heavy
lua user) one of the key advantages of Ethos.

Please note events like throttle cut, throttle active, telemetry lost seem
to hold their status as expected.

Thanks again.

**Comment by Jedsters** [comment]
This sounds similar to #3882

**Comment by bsongis-frsky** [comment]
Fix ready for the next nightlies

**Comment by Alex00101** [comment]
May I please ask if this fix made it into latest 1.5.8?

Thanks

**Comment by bsongis-frsky** [comment]
Release note updated

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/78dafb5a-fa10-47b5-ba11-868d320b8097)

**Comment by Alex00101** [comment]
Thanks a lot!
